arto-converter/artoConverter.cpp

```diff
@@ -112,6 +112,85 @@ bool convert_arabic_to_roman(unsigned int arabic_number, char* roman_num)
   return true;
 }
 
+////////////////////////////////////////////////////////////////////////////////////
+//----------------------------------------------------------------------------------
+
+#define ILL_TERMINATOR  " ill"
+#define AMB_TERMINATOR  " amb"
+#define ERR_TERMINATOR  " err"
+#define AMB_DIGIT       -1
+#define AMB_SYMBOL      '?'
+#define MAX_CHECK_SUM   11
```

Comment:
```suggestion
#define CONST_CHECK_SUM   11
```
Предлагаю такое название

Reply:
Возможно так будет лучше 👍 
```suggestion
#define MAGIC_CHECKSUM_CONST   11
```